app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -1,4 +1,6 @@
 require 'csv'
+require 'reporting/command_line_options'
```

Comment:
we shouldn't need the `command_line_options` in this file, that we should update the `monthly_proofing_report` file to require that if it's missing